pom.xml

```diff
@@ -128,6 +129,18 @@
                 <version>${version.com.google.guava.guava}</version>
             </dependency>
             <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <scope>provided</scope>
+                <version>${version.org.apache.httpcomponents.httpclient}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpcore</artifactId>
+                <version>${version.org.apache.httpcomponents.httpcore}</version>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
That's not a problem, though, no? We're not using the `httpcore` APIs directly, are we?